easybuild/easyconfigs/g/GPAW/GPAW-1.3.0-enable-scalapack-and-libs.patch

```diff
@@ -0,0 +1,78 @@
+--- ../gpaw-1.3.0-82cebebc037510d876f90d9f8d533fd021f751f5/customize.py	2017-10-02 12:41:22.000000000 +0200
```

Comment:
@martbhell Please include short description of the patch above this line, and mention author or upstream source

Reply:
Also, did you take a look at @migueldiascosta's suggestions on this patch in https://github.com/easybuilders/easybuild-easyconfigs/pull/5613#pullrequestreview-111068948?